src/main/scripts/org/reaktivity/specification/nukleus/http_cache/streams/proxy/edge-arch/maintain.polling.per.multiple.auth.scopes/accept/client.rpt

```diff
@@ -0,0 +1,96 @@
+#
```

Comment:
John wants us to prefer the term realm to scope since it's more standard, so suggest rename to ...for.multiple.auth.realms (we could even drop the .auth part perhaps since realm is really only used in that sense)